main.go

```diff
@@ -159,14 +287,29 @@ func main() {
 		for {
 			totalLoaded = 0
 			t1 := time.Now()
-			fmt.Printf("Checking %v wallets...\n", len(allWatching))
+			fmt.Printf("Checking %v wallets and %v contracts...\n", len(allWatching), len(allContracts))
+
+			// Check wallet balances
 			for _, v := range allWatching {
 				v.Balance = GetEthBalance(v.Address).String()
 				totalLoaded++
 			}
+
+			// Check contract functions
+			for _, c := range allContracts {
+				result, err := CallContractFunction(c.Address, c.ABI, c.Function)
+				if err != nil {
+					fmt.Printf("Error calling contract function %s on %s: %v\n", c.Function, c.Name, err)
+					c.Result = "0"
```

Comment:
It may be interpreted as a false-positive result. Wouldn't it be better to set an empty string?

```suggestion
					c.Result = ""
```

Reply:
[d02c63f](https://github.com/mezo-org/balance-exporter/pull/1/commits/d02c63fe19c1112deaf6ad9d9f2341f92cceebd0)